docs/source/guide/index.md

```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
+**Label studio enterprise**
+Label Studio Enterprise is a data labeling platform that enables you to build ground truth datasets, with the focus on quality, security, and team collaboration. The only end-to-end solution for managing internal data labeling projects.
+
+**Lightweight Directory Access Protocol (LDAP)**
+LDAP is a mature, flexible, and well-supported standards-based mechanism for interacting with directory servers. It is often used for authentication and storing information about users, groups, and applications, but an LDAP directory server is a fairly general-purpose data store and can be used in a wide variety of applications.
+
+**Machine learning**
+Machine learning is a branch of AI and computer science which focuses on the use of data and algorithms to imitate the way that humans learn, gradually improving its accuracy.
+
+**Machine Learning Model Operationalization Management (MLOps)**
+MLOps is a set of practices that aims to deploy and maintain machine learning models in production reliably and efficiently.
+
+**Model**
+A model in machine learning is the output of a machine learning algorithm run on data. A model represents what was learned by a machine learning algorithm.
+
+**Modeling**
+The process of modeling means training a machine learning algorithm to predict the labels from the features, tuning it for the business need, and validating it on holdout data.
+
+**Named entity recognition (NER)**
+NER is the process of identifying and classifying named entities presented in a text document.
+
+**Natural language processing (NLP) labeling**
+Data labeling is the process of adding labels to raw data to give context or meaning to the data. The labeled data is then used to train the NLP models to make predictions or understand or generate speech.
+
+**Object detection**
+Object detection is the task of detecting instances of objects of a certain class within an image.
+
+**Object detection dataset**
+Object detection relates to computer vision and image processing that deals with detecting instances of semantic objects of a certain class (such as humans, buildings, or cars) in digital images and videos.
+
+**Optical Character Recognition (OCR)**
+Optical character recognition (OCR) is a technology that analyzes the text of a page and turns the letters into code that may be used to process information.
+
+**Outliner**
+Outliner is the place where you can see all the details about annotation, regions, and labeling history. The Outliner and Details panel are both collapsible and detachable, so you can arrange them the way you want.
+
+**Persistent storage**
+Persistent storage is any data storage device that retains data after power to that device is shut off. It is also sometimes referred to as nonvolatile storage.
+
+**PostgreSQL**
+PostgreSQL, commonly pronounced “Post-GRE” is an open source database that has a strong reputation for its reliability, flexibility, and support of open technical standards.
+
+**Predictions** 
```

Comment:
In LS predictions and pre-annotations are the same, better to combine it with pre-annotations